tools/run_tests/performance/scenario_config.py

```diff
@@ -266,6 +266,13 @@ def scenarios(self):
                   server_type='%s_SERVER' % synchronicity.upper(),
                   unconstrained_client=synchronicity, secure=secure,
                   categories=[SWEEP], channels=channels, outstanding=outstanding)
+              yield _ping_pong_scenario(
+                  'cpp_protobuf_%s_streaming_qps_unconstrained_%s_%d_channels_%d_outstanding' % (synchronicity, secstr, channels, outstanding),
+                  rpc_type='STREAMING',
```

Comment:
Should rpc_type be an iterator also, just like synchronicity? This will shave off some lines of code above as well. For protobuf tests we are now testing everything with unary and streaming, sync and async.